src/module/Ui/templates/layout/en/serlo-home.phtml

```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
     <div id="page">
         <section class="clearfix">
             <div id="content-layout" class="clearfix">
-                <?php echo $this->partial('layout/partials/notifications'); ?>
-
-                <div class="home-subject-row home-row">
-                    <div class="home-column">
-                        <a href="/23591">Learn maths <i class="fa fa-arrow-circle-right"></i></a><br/>
-                    </div>
+                <div class="flasher">
                 </div>
 
                 <div class="home-serlo-row home-row">
                     <div class="home-column">
                         <div class="logo-wrapper">
-                            <?php echo $this->brand()->getLogo(); ?>
```

Comment:
Why did you remove this? Isn't this equivalent to `<span class="serlo-logo">V</span> <div class="serlo-brand">Serlo</div> </div>`?